src/include/daos_srv/rebuild.h

```diff
@@ -35,6 +35,36 @@ typedef enum {
 			  (rb_op) == RB_OP_NONE ? "None" : \
 			  "Unknown")
 
+/* Common rebuild identifying information for INFO/DEBUG logging:
+ * rb=<pool_uuid>/<rebuild_ver>/<rebuild_gen>/<opstring>
+ */
+#define DF_RB  "rb=" DF_UUID "/%u/%u/%s"
+
+/* Full rebuild identifying information includes <term>/<rebuild_gen> */
```

Comment:
That is correct. DF_RBF was intented to include leader_rank as the final value, and I missed updating the comment.

If I remove leader_rank here, then the remaining information would be the PS term. Shall we keep the term in the DF_RBF rebuild log messages? If not, then there is no need for DF_RBF at all.

For now, until I see your response, I will update the patch so DF_RBF will not include the leader_rank, and keep the term value.

Reply:
I think need not log the leader rank/term in every log entry, we may only log that info when rebuild code detect leader switched, for example in rebuild_tgt_scan_handler() it found "rpt->rt_leader_rank != rsi->rsi_master_rank".
So seems that can keep DF_RBF and still with leader_rank and leader_term, but only use it in special place with leader switch, rather than add it to all/most log entries. What do you think? thanks for considering it.

Reply:
Yes, let's start with this approach, minimizing its use, and see if we may find other usages for DF_RBF later. I've updated DF_RBF and usage sites in the latest push.